tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+	"compilerOptions": {
+		"allowJs": true,
+		"allowSyntheticDefaultImports": true,
+		"checkJs": true,
+		"jsx": "preserve",
+		"lib": ["dom", "esnext"],
+		"module": "commonjs",
+		"noEmit": true,
+		"resolveJsonModule": true,
+		"target": "esnext",
+
+		/* Strict Type-Checking Options */
+		"strict": true,                        /* Enable all strict type-checking options. */
+		"noImplicitAny": false,                /* Raise error on expressions and declarations with an implied 'any' type. */
+
+		/* Additional Checks */
+		"noUnusedLocals": true,                /* Report errors on unused locals. */
+		"noUnusedParameters": true,            /* Report errors on unused parameters. */
+		"noImplicitReturns": true,             /* Report error when not all code paths in function return a value. */
+		"noFallthroughCasesInSwitch": true,    /* Report errors for fallthrough cases in switch statement. */
+
+	},
+	"include": [
+		"./packages/url/**/*.js"
+	],
+	"exclude": [
+		"./packages/**/test/**",
```

Comment:
At some point, you will have to include:
- `"./packages/**/benchmark/**",`

Just saying, there are a few packages which use them :)